src/js/jsx/sections/export/LayerExports.jsx

```diff
@@ -0,0 +1,297 @@
+        },
+        "2": {
+            id: "2",
+            title: "2x"
+        }
+    });
+
+    var formatOptions = Immutable.OrderedMap({
+        "png": {
+            id: "png",
+            title: "PNG"
+        },
+        "jpg": {
+            id: "jpg",
+            title: "JPG"
+        },
+        "svg": {
+            id: "svg",
+            title: "SVG"
+        },
+        "pdf": {
+            id: "pdf",
+            title: "PDF"
+        }
+    });
+
+    var allScales = [0.5, 1, 1.5, 2];
+
+    var LayerExportAsset = React.createClass({
+
+        mixins: [FluxMixin],
+
+        propTypes: {
+            index: React.PropTypes.number.isRequired,
+            layer: React.PropTypes.object.isRequired,
+            exportAsset: React.PropTypes.object.isRequired
+        },
+
+        /**
+         * Delete this asset
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleDeleteClick: function () {
+            var document = this.props.document,
+                layer = this.props.layer,
+                index = this.props.index;
+
+            this.getFlux().actions.export.deleteLayerExportAsset(document, layer, index);
+        },
+
+        /**
+         * Update this asset's scale
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleUpdateScale: function (scale) {
+            var scaleNum = Number.parseFloat(scale);
+
+            this.getFlux().actions.export.updateLayerAssetScale(
+                this.props.document, this.props.layer, this.props.index, scaleNum);
+        },
+
+        /**
+         * Update this asset's suffix
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleUpdateSuffix: function (event, suffix) {
+            this.getFlux().actions.export.updateLayerAssetSuffix(
+                this.props.document, this.props.layer, this.props.index, suffix);
+        },
+
+        /**
+         * Update this asset's format
+         *
+         * @private
+         * @return {Promise}
+         */
+        _handleUpdateFormat: function (format) {
+            var formatLower = format && format.toLowerCase();
+
+            this.getFlux().actions.export.updateLayerAssetFormat(
+                this.props.document, this.props.layer, this.props.index, formatLower);
+        },
+
+        render: function () {
+            var layer = this.props.layer,
+                exportAsset = this.props.exportAsset,
+                scale = exportAsset.scale || 1,
+                scaleOption = scaleOptions.has(scale.toString()) ?
+                    scaleOptions.get(scale.toString()) : scaleOptions.get("1"),
+                scaleListID = "layerExportAsset-scale" + layer.id + "-" + this.props.index,
+                formatListID = "layerExportAsset-format-" + layer.id + "-" + this.props.index;
+
+            return (
+                <div className="formline">
+                    <Datalist
```

Comment:
So we aren't going to allow for arbitrary numbers to be entered here? If this is is because @placegraphichere's comps showed widget that allowed you to enter arbitrary numbers but also had a menu for default selections, we've decided elsewhere that for the first pass this should just be a number input. But, maybe in this case we _only_ want to allow the user to choose from a finite set of scale options. In any case, we should explicitly choose one or the other for the near term.